File: src/Game2.java
import java.util.Scanner;

public class Game2 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.println("Player 1 input a character, this must be: P,H,F");
        char p1 = sc.next().charAt(0);
        p1 = Character.toUpperCase(p1);
        System.out.println("Player 2 input a character, this must be: P,H,F");
        char p2 = sc.next().charAt(0);
        p2 = Character.toUpperCase(p2);

        if (p1 == 'P') {
            if (p2 == 'P') {
                System.out.println("Draw");
            } else if (p2 == 'F') {
                System.out.println("Player 2 won! Scissors cut paper");
            } else if (p2 == 'H') {
                System.out.println("Player 1 won! Paper covers rock");
            } else {
                System.out.println("Invalid Character at player 2");
            }
        } else if (p1 == 'F') {
            if (p2 == 'F') {
                System.out.println("Draw");
            } else if (p2 == 'P') {
                System.out.println("Player 1 won! Scissors cut paper");
            } else if (p2 == 'H') {
                System.out.println("Player 2 won! Rock beats scissors");
            } else {
                System.out.println("Invalid Character at player 2");
            }
        } else if (p1 == 'H') {
            if (p2 == 'H') {
                System.out.println("Draw");
            } else if (p2 == 'F') {
                System.out.println("Player 1 won! Rock beats scissors");
            } else if (p2 == 'P') {
                System.out.println("Player 2 won! Paper covers rock");
            } else {
                System.out.println("Invalid Character at player 2");
            }
        } else {
            System.out.println("Invalid Character at player 1");
        }
    }
}

File: src/Game2.py
p1 = input("Player 1 input a character, this must be: P, H, F\n").upper()
p2 = input("Player 2 input a character, this must be: P, H, F\n").upper()

valid = {'P', 'H', 'F'}

if p1 not in valid:
    print("Invalid Character at player 1")
elif p2 not in valid:
    print("Invalid Character at player 2")
elif p1 == p2:
    print("Draw")
elif p1 == 'P':
    if p2 == 'F':
        print("Player 2 won! Scissors cut paper")
    else:
        print("Player 1 won! Paper covers rock")
elif p1 == 'F':
    if p2 == 'P':
        print("Player 1 won! Scissors cut paper")
    else:
        print("Player 2 won! Rock beats scissors")
elif p1 == 'H':
    if p2 == 'F':
        print("Player 1 won! Rock beats scissors")
    else:
        print("Player 2 won! Paper covers rock")
